addons/compat_spe/compat_spe_csw/CfgMagazineGroups.hpp

```diff
@@ -0,0 +1,98 @@
+class EGVAR(csw,groups) {
+    // --- Gun Turrets -------------------------------------------------------------
+    class SPE_50Rnd_762x63 {
+        SPE_50Rnd_762x63 = 1;
+    };
+    class SPE_100Rnd_762x63 {
+        SPE_100Rnd_762x63 = 1;
+    };
+
+    class SPE_50Rnd_762x63_M1 {
+        SPE_50Rnd_762x63_M1 = 1;
+    };
+    class SPE_100Rnd_762x63_M1 {
+        SPE_100Rnd_762x63_M1 = 1;
+    };
+
+    class SPE_50Rnd_762x63_M2_AP {
+        SPE_50Rnd_762x63_M2_AP = 1;
+    };
+    class SPE_100Rnd_762x63_M2_AP {
+        SPE_100Rnd_762x63_M2_AP = 1;
+    };
+
+    class SPE_50Rnd_762x63_Tracer {
+        SPE_50Rnd_762x63_Tracer = 1;
+    };
+    class SPE_100Rnd_762x63_Tracer {
+        SPE_100Rnd_762x63_Tracer = 1;
+    };
+
+    class SPE_50Rnd_792x57 {
+        SPE_50Rnd_792x57 = 1;
+    };
+    class SPE_100Rnd_792x57 {
+        SPE_100Rnd_792x57 = 1;
+        SPE_250Rnd_792x57 = 1;
+    };
+
+    class SPE_50Rnd_792x57_sS {
+        SPE_50Rnd_792x57_sS = 1;
+    };
+    class SPE_100Rnd_792x57_sS {
+        SPE_100Rnd_792x57_sS = 1;
+        SPE_250Rnd_792x57_sS = 1;
+    };
+
+    class SPE_50Rnd_792x57_SMK {
+        SPE_50Rnd_792x57_SMK = 1;
+    };
+    class SPE_100Rnd_792x57_SMK {
+        SPE_100Rnd_792x57_SMK = 1;
+        SPE_250Rnd_792x57_SMK = 1;
+    };
+
+    class SPE_50Rnd_792x57_Tracer {
+        SPE_50Rnd_792x57_Tracer = 1;
+    };
+    class SPE_100Rnd_792x57_Tracer {
+        SPE_100Rnd_792x57_Tracer = 1;
+    };
+
+    /*
+    // Makes it compatible with other mods' .50 BMG
+    class EGVAR(csw,100Rnd_127x99_mag) {
+        SPE_100Rnd_127x99_M2 = 1;
+    };
```

Comment:
I'm hesitant to add this, as if you load a `SPE_100Rnd_127x99_M2` magazine into a CSW, it will be converted into a `EGVAR(csw,100Rnd_127x99_mag)` when unloaded.

Reply:
Better to convert it, to have a maximum of compatibility possible.